### PR TITLE
doc: add minutes for meeting Oct 25 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-10-25.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+## Announcements
+
+* Security release yesterday.  There was an unrelated regression in 8.8.0 release, 8.8.1 release
+   being prepared today. Includes unflagging HTTP2.
+
+### WG update
+
+* Update from the moderation team (Forrest)
+
+* Deck used for persentation https://docs.google.com/presentation/d/1cN712pneaa7L6_wVl9-M2OpK_2Am8EbarYyoJxxyyYA/edit#slide=id.p
```
Comment (maclover7) [inline meetings/2017-10-25.md]: presentation

File: meetings/2017-10-25.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  as to process for this (since they need to implement it).
+* Forrest, as long as policy is clear enough, it should be able to be applied to
+  leaders and members alike.  
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Support both OpenSSL 1.1.0 and 1.0.2 [#16130](https://github.com/nodejs/node/pull/16130)
+  * current state is that is is getting ready to land
```
Comment (maclover7) [inline meetings/2017-10-25.md]: it is?

File: meetings/2017-10-25.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2017-10-25.md]: fixed.

File: meetings/2017-10-25.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2017-10-25.md]: fixed

**Comment by mhdawson** [comment]
Pushed changes to address comments.  @nodejs/tsc please review.

File: meetings/2017-10-25.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+* Security release yesterday.  There was an unrelated regression in 8.8.0 release, 8.8.1 release
+   being prepared today. Includes unflagging HTTP2.
+
+### WG update
+
+* Update from the moderation team (Forrest)
+
+* Deck used for presentation https://docs.google.com/presentation/d/1cN712pneaa7L6_wVl9-M2OpK_2Am8EbarYyoJxxyyYA/edit#slide=id.p
+
+* 10 members, so far have had to be intervene in 0 zeros, though there were 2 request
```
Comment (cjihrig) [inline meetings/2017-10-25.md]: What is 0 zeros? Also, request -> requests.

File: meetings/2017-10-25.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+   for guidance
+* team not working group
+* scope set as enforcing policies, not setting them.
+* With thefourtheye joining the team now has global timezone coverage
+  (this has been   challenge due to being a small team)
```
Comment (cjihrig) [inline meetings/2017-10-25.md]: There are a few extra spaces after "been"

File: meetings/2017-10-25.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  * Michael, agree suggested PR to start discussion on them.  Don.t believe this should
+    block the expectations document.
+  * Myles doing additional values session and would like to avoid landing things
+    until we get that feedback.  Does not believe it should block expectations doc
+  * William mentioned current version has removed some references to values and
+    believes we should not block either.
+
+## Q&A, Other
+
+* From irc, Bryan, number of uncharted WG's not an issue to have groups to
```
Comment (cjihrig) [inline meetings/2017-10-25.md]: Is it worth adding Bryan's last name? I'm not sure that it matters.

File: meetings/2017-10-25.md (hunk 3 above)
Comment (mhdawson) [reply meetings/2017-10-25.md]: Fixed, believe if should have been issues.  Added s to request.

File: meetings/2017-10-25.md (hunk 4 above)
Comment (mhdawson) [reply meetings/2017-10-25.md]: Fixed

**Comment by mhdawson** [comment]
Landed as ce1e60b08984aac4804c5c8a92a9fc6b50196fb7

**Comment by mhdawson** [comment]
Landed as b585c06b89d54b9109b8b2afc12b04c5c0fa3dac

**Pull request closed without merging**